UndertaleModTool/Scripts/Repackers/ImportShaderData.csx

```diff
@@ -1,33 +1,23 @@
 using System.Text;
 using System;
+using System.Linq;
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
 
 EnsureDataLoaded();
 
-// "Select 'Import_Loc.txt' file in 'Shader_Data'"
 string importFolder = PromptChooseDirectory();
 if (importFolder == null)
     throw new ScriptException("The import folder was not set.");
 
-string[] dirFiles = Directory.GetFiles(importFolder, "*.*", SearchOption.AllDirectories);
-List<string> shadersToModify = new List<string>();
+var shadersToModify = Directory.GetDirectories(importFolder).Select(x => Path.GetFileName(x));

```

Comment:
This does not seem to be the equivalent of the foreach loop from before.

Reply:
Yes, because the algorithm is changed completely.
This approach is better.

Reply:
Double checked, see now.